### PR TITLE
add relavant wum timestamp

### PR DESCRIPTION
**Pull request by isharac**
## Purpose
update the doc with wum timestamp up to which the provided connector is valid.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md [hunk 1]
```diff
@@ -143,7 +143,9 @@ Follow the instructions below to set up and configure the databases for the WSO2
 
 ## Step 4 - Configure WSO2 IS with WSO2 API-M
 
-1. Download the [WSO2 IS Connector]({{base_path}}/assets/attachments/administer/wso2is-km-connector-1.0.16.zip).
+1. Download the WSO2 IS Connector based on the API-M version.
+     - [WSO2 IS Connector for GA Relealse]({{base_path}}/assets/attachments/administer/wso2is-km-connector-1.0.16_ga.zip).
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md:147]: Spelling mistake
"Relealse" should be "Release"

File: en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md [hunk 2]
```diff
@@ -143,7 +143,9 @@ Follow the instructions below to set up and configure the databases for the WSO2
...
+     - [WSO2 IS Connector for WUM updated version(as at 1611933891844)]({{base_path}}/assets/attachments/administer/wso2is-km-connector-1.0.16.zip).
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md:148]: Why are we mentioning the WUM timestamp? Thus far we have always quoted the WUM release date when mentioning WUM features.

@tharindu1st , please clarify.

**Review by Mariangela — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md (hunk 2 above)
Comment (isharac) [reply en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md:148]: ack. will change to the date

File: en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md (hunk 1 above)
Comment (isharac) [reply en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md]: Fixed with 11b0ccf3681d4452e788beee35119a7ae618852d

File: en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md (hunk 2 above)
Comment (isharac) [reply en/docs/install-and-setup/setup/distributed-deployment/configuring-wso2-identity-server-as-a-key-manager.md]: Fixed with 11b0ccf3681d4452e788beee35119a7ae618852d

**Review by Mariangela — APPROVED** [review]

**Pull request merged**